pkg/gui/views.go

```diff
@@ -223,6 +253,7 @@ func (gui *Gui) createAllViews() error {
 	gui.Views.Extras.FgColor = theme.GocuiDefaultTextColor
 	gui.Views.Extras.Autoscroll = true
 	gui.Views.Extras.Wrap = true
+	gui.Views.Extras.FrameRunes = frameRunes
```

Comment:
A cleaner approach would be to, at the bottom of this method, iterate through each view and if `Frame` is true, set FrameRunes appropriately. We could even just set it unconditionally given that FrameRunes is ignored for un-framed views.

Lemme know if you need help with that

Reply:
Once set only if Frame is true at the end
But if you want to set it unconditionally, I think you can set it in the first loop
Which would be better?

Reply:
@jesseduffield 
I decided to set it unconditionally because it will have one loop and I don't have to use refrect.